src/Apps/gNtpConv.cxx

```diff
@@ -398,6 +398,11 @@ void ConvertToGST(void)
                                  //  - ((e/h) * energy)   for pi0, gamma, e-, e+, where e/h is set to 1.3
                                  //  - (kinetic energy) for other particles
 
+  double brXSec;              // the event cross section in 1E-38cm^2
+  double brDXSec;             // is the differential cross section for the selected in 1E-38cm^2/{K^n}
+  int    brKPS;               // phase space that the xsec has been evaluated into as defined in
```

Comment:
I would prefer this to be an `unsigned int`. So, I would prefer this to be `UInt_t`